dot/state/service.go

```diff
@@ -321,13 +322,22 @@ func (s *Service) Start() error {
 	return nil
 }
 
-// Rewind rewinds the chain by the given number of blocks.
+// Rewind rewinds the chain to the given block number.
 // If the given number of blocks is greater than the chain height, it will rewind to genesis.
-func (s *Service) Rewind(numBlocks int) error {
+func (s *Service) Rewind(to int64) error {
```

Comment:
Rename the variable `to` to `toBlockNumber` or something more meaningful.

Reply:
renamed to `toBlock`